boa3/model/builtin/classmethod/tobytesmethod.py

```diff
@@ -75,12 +75,46 @@ def __init__(self, self_type: IType = None):
         super().__init__(self_type)
 
     def build(self, value: Any) -> IBuiltinMethod:
+        if isinstance(value, List) and len(value) == 1:
```

Comment:
```suggestion
        if isinstance(value, Sequence) and len(value) == 1:
```
so it can value both `list` and `tuple`